lib/ret_web/channels/hub_channel.ex

```diff
@@ -238,6 +238,23 @@ defmodule RetWeb.HubChannel do
     {:noreply, socket}
   end
 
+  def handle_in("get_perms_token", _args, socket) do
```

Comment:
i'm using `refresh_perms_token` on my branch for the global refresh one on `ret_channel`, i defer to you which name you prefer